system/api/create.go

```diff
@@ -29,19 +31,43 @@ type Trustable interface {
 	AutoApprove(http.ResponseWriter, *http.Request) error
 }
 
+func jsonToURLValues(data *[]byte) (*url.Values, error) {
```

Comment:
I'm returning pointer here to avoid coping, but I'm not sure if go will copy values or move it